packages/nextjs/hooks/scaffold-stark/useScaffoldReadContract.ts

```diff
@@ -28,15 +28,11 @@ export const useScaffoldReadContract = <
     address: deployedContract?.address,
     abi: deployedContract?.abi,
     watch: true,
-    args,
-    enabled: !Array.isArray(args) || !args.some((arg) => arg === undefined),
+    args: args ?? [],
+    enabled: true,
```

Comment:
might need to be careful with enabling everything right on the start, because this will immediately fetch the contract. my suggestion is to default to `true` and then allow the user to override? wdyt @0xquantum3labs

Reply:
I agree with you, we are able to `override` this flag right now.